### ISSUE TITLE
Answer feedback

### ISSUE THREAD

**Issue by richelbilderbeek** [richelbilderbeek/travis_cpp_tutorial#28]
Some awesome feedback I got by email from `flywire`:

> I'm interested in setting up Travis CI for LibreCad V3, a c++ program. Your tutorial (and blog etc) should be the ideal intro as it's the same environment but I'm having trouble following it. I'd like to provide feedback and ideally contribute to addressing these issues. (Maybe you're not so interested in this old project now.)
> 
> I'm using https://github.com/richelbilderbeek/travis_cpp_tutorial/blob/master/travis_cpp_tutorial.pdf
> 
>  From Travis to source
> 
> 
>  So right from the start:
> 
>  * 2.1 create https://github.com/flywire/cold_fusion_reactor
>  * 2.2 git clone https://github.com/richelbilderbeek/travis_cpp_tutorial
>  * 2.3 Create a Qt Creator project
> 
>     Why did we go from the cold_fusion_reactor repository to https://github.com/richelbilderbeek/travis_cpp_tutorial ?

I think @flywire has a point and I do not connect the chapters of the tutorial well enough.

>     What is Qt? (I know what it is but I don't see how it fits into the context of the tutorial.)
> 
> So I haven't got going yet but I can see from the code snippets this tutorial contains exactly what I'm after.
> 
> I find the font in the pdf to small to view on one side of a 22" monitor as I work through the tutorial. (I guess I'm getting spoilt with mkdocs.) 

I see the point. As can be seen [here](https://github.com/richelbilderbeek/travis_cpp_tutorial/blob/master/tutorial.md), I tried to do a proper conversion, but fail for now. If anyone know how to properly run the script, we can adapt it to people's need. Now, I think I just used the LyX default setting.

> btw, where does https://github.com/richelbilderbeek/simplify_cpp_travis_intro fit in?

AFAICS, I do not mention this in the tutorial. The tutorial is far from finished and I see what I intend it to become. The [https://github.com/richelbilderbeek/simplify_cpp_travis_intro](https://github.com/richelbilderbeek/simplify_cpp_travis_intro) is a general article.

> Hopefully you'll take these comments as constructive criticism as they are intended.

I definitely am! Thanks and please send more if needed :+1:

**Comment by flywire** [comment]
I have a better idea of the process the tutorial is trying to explain now. The contents table reads really well. The issue is more than not connecting the chapters, the initial steps don't actually flow to build the environment.

Given (after checking travis_cpp_tutorial.lyx) `3 The basic build` starts with https://github.com/richelbilderbeek/travis_qmake_gcc_cpp98 ( then the previous section should work towards setting up the environment for the project. Don't confuse things with different repository names etc.

It is not clear where the user types what to establish the repository. Does the user just clone the existing repository or do they setup a Qt environment and use git to populate it?

Users will have a range of existing understanding and it is hard following something explaining things you already understand well. Consider starting the tutorial with a fully built project like  https://github.com/richelbilderbeek/simplify_cpp_travis_intro. (I'm expecting a Hello_World.exe for windows out of this.) Users could then work through the tutorial or use it as a reference to clarify parts they don't understand.

**Comment by richelbilderbeek** [comment]
I agree completely! As we agree, I will happily add you as a Collaborator if you want! For me, this tutorial -up until now- has been mostly a place to put some placeholder texts. If you'd enjoy to help, I'd happily let you!

**Comment by flywire** [comment]
> I will happily add you as a Collaborator

OK. I think by the time I work through the tute it should be pretty reasonable.

I'll need to work with the doc and iirc you use a mac. I don't have any experience with, or a particular interest in, LyX which I suppose you are familiar with from an academic background. So my preference would be some other format, maybe mkdocs, reStructuredText or a commented docx, but if LyX is easy enough I suppose I could work with it.

The environment details probably need to go into the wiki. What do you think?

**Comment by richelbilderbeek** [comment]
> and iirc you use a mac

Nah, I use Linux for at least a decade now :nerd_face: :+1:

LyX is easy (I do use LaTeX nowadays) and I am open to any open cross-platform format. `mkdocs` appears great to me. I do wonder if it wouldn't be too much work for you! If not, I'll definitely let you help!

Thanks so much for your feedback!

**Comment by richelbilderbeek** [comment]
P.S. If you want, we can do a videochat. It's common nowadays to do so :+1:

**Comment by flywire** [comment]
Ok, we'll go with mkdocs and maybe move it to reStructuredText if needed. I'm on Win10. How about Discord?

Are you happy to present the end at the start? We could just tell people to read the last chapter to see where it is headed and then come back to work through or skim as required to finish the tute. Other than that perhaps make it clearer where the user types what and how to move from one part to the next.

I'll have to check out how to include code files directly.

**Comment by richelbilderbeek** [comment]
Hi @flywire: sure, Discord would work fine, my username is `richelbilderbeek#9002`. I do not know your timezone, but I am available tomorrow/Saturday from 12:00-18:00 Amsterdam time zone. Let me know if I am lucky and you can meet me there as well :+1:

**Comment by flywire** [comment]
Should be a discord invite in your email.

**Comment by richelbilderbeek** [comment]
Thanks! Looking forward to it!

**Comment by richelbilderbeek** [comment]
> 4:30 PM] richelbilderbeek: except that the Qt Creator part needs a better explanation
> [4:31 PM] flywire: I'd like it to take a peek at the last chapter at the start of chapter 3 to explain where it is headed.

Will do so Saturday June 13th :+1:

**Comment by richelbilderbeek** [comment]
I worked on this already Friday June 12th. It was more work than expected to get the LyX doc working. I will continue next week, Saturday June 20th :+1:
